Dockerfile

```diff
@@ -1,3 +1,9 @@
+FROM node:8.9 as buildenv
```

Comment:
Update this to the latest version of node 10 docker has available while we're at it.